R/import_data.R

```diff
@@ -1,29 +1,29 @@
 #' Import raw data
 #'
 #' @param manifest_file character, file path(s) to manifest(s)
-#' @param template_file data.frame, with datapaths and names of results file(s)
+#' @param template_file character, with datapaths and names of results file(s)
 #' or character with file path of templates file(s)
-#' @param results_file data.frame, with datapaths and names of results file(s)
+#' @param results_file character, with datapaths and names of results file(s)
 #' or character with file path of results file(s)
-#' @param instrument character with type of instrument used
+#' @param instrument string with type of instrument used
 #' 
 #' @examples
 #' td <- get_test_data()
 #' i_df <- import_data(manifest_path(td), template_path(td), result_path(td))
 #' 
-#' @return a \code{data.frame}
+#' @return a \code{data.table}
 #'
 #' @export
 #'
 import_data <- function(manifest_file,
                         template_file,
                         results_file,
                         instrument = "EnVision") {
-  loaded_data <- load_data(manifest_file = manifest_file,
-                           df_template_files = template_file,
-                           results_file = results_file,
-                           instrument = instrument)
-  merge_data(manifest = loaded_data$manifest,
-             treatments = loaded_data$treatments,
-             data = loaded_data$data)
+  loaded_data <- gDRimport::load_data(manifest_file = manifest_file,
```

Comment:
Namespacing is not required: https://github.com/gdrplatform/gDR/blob/ef48d2681079c21f96dd2a746573c06d0116d69e/R/package.R#L3-L4